test/CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ endfunction()
 # Include generated files (moc ouput, build configuration)
 include_directories("${CMAKE_CURRENT_BINARY_DIR}/../src")
```

Comment:
For consistency with the line inserted below, `include_directories("${PROJECT_BINARY_DIR}/src")` ?